Js Session-1/js/script.js

```diff
@@ -0,0 +1,95 @@
+// on load hide field od showcity
+function init() {
+	document.getElementById('selectCity').style.display='none';	
+}
+
+// to change screen on anchor
+function changeScreen(target) {
+	    window.location.href = target+'.html';
+}
+
+$(document).ready(function(){
+    $('#mylist a').click(function(){
+        switch ($('#mylist a').index(this)) {
+			case 0: window.location.href = 'index.html';
+					break;
+			case 1:	alert("No page yet");
+				    break;
+			case 2:	alert("No page yet");
+					break;
+			case 3:	alert("No page yet");
+					break;
+			case 4: window.location.href = 'contact.html';
+					break;			
+		}
+    });
+});
+
+/* to validate name
+   min char -> 5
+   max char -> 20   
+*/
+function validateName(nameField) {
+    var regex = /^[a-zA-Z ]{5,20}$/;
+    if (!regex.test(nameField.value)) {
+        nameField.style.borderColor="red";
+        return false;
+    }
+    nameField.style.borderColor="#DCDCDC";
+	return true;
+}
+
+// to validate email
+function validateEmail(emailField) {
+	var reg = /^([A-Za-z0-9_\-\.])+\@([A-Za-z0-9_\-\.])+\.([A-Za-z]{2,4})$/;
```

Comment:
You should have write only one method like this:
function validateForm(regEx, field)
then only single method was sufficient, no need to write separate method for each field. From HTML page pass regEx and field from field that you want to validate. Discuss this with me if not clear.